packages/react/ds/src/tabs/tab-item.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import { slugify } from '../utils.js';
+import { Link } from '../link/link.js';
+
+export function TabItem({
+  value,
+  href,
+  checked = false,
+  children,
+}: {
+  value: string;
+  href: string;
```

Comment:
Intended to be a required prop? 

Should the link become "link" colour? 
<img width="278" alt="image" src="https://github.com/user-attachments/assets/0047465d-278d-4b77-a436-dac02ae872b1">

Does anybody possibly need to know what is active based on a click, or is this supposed to be a readonly component (makes sense). 

There's a behaviour I'm not convinced on, if you provide a href, you can click slightly off the link text and it will only change tab, not hit the anchor.